framework_lib/src/commandline/clap_std.rs

```diff
@@ -155,6 +155,16 @@ struct ClapCli {
     #[arg(long)]
     charge_limit: Option<Option<u8>>,
 
+    /// Get or set max charge current limit
+    #[arg(long)]
+    #[clap(num_args = ..2)]
+    charge_current_limit: Vec<u32>,
+
+    /// Get or set max charge current limit
+    #[arg(long)]
+    #[clap(num_args = ..2)]
```

Comment:
If I'm understanding correctly, I believe this limits the argument count to a max of 1, and if you want to allow 2, you need `..=2` instead, right?  I mention this because I'm trying to use framework_tool v0.4 with `--charge-rate-limit 60 0.7` (similar to the syntax given in the new examples) but I get a clap error, "unexpected argument '0.7' found".  It does work with a single argument, but I wanted to use the conditional form with the second argument based on battery state of charge.

Reply:
Yes, you're right. Thanks!. https://github.com/FrameworkComputer/framework-system/pull/123